### PR TITLE
add two FAQs re generating of UID2 values

### PR DESCRIPTION
**Pull request by genwhittTTD**
add two FAQs re generating of UID2 values

### CODE REVIEW

File: docs/getting-started/gs-faqs.md [hunk 1]
```diff
@@ -185,6 +194,12 @@ When a user opts out of UID2-based targeted advertising through the [Transparenc
 
 If a user opts out through your website, you should follow your internal procedures for handling the opt-out. For example, you might choose not to generate a UID2 for that user.
 
+#### Does the same DII always result in the same raw UID2?
+xxx
+In general yes, the process of generating a raw UID2 from DII is the same, and results in the same value, no matter who sent the request. If two UID2 participants were to send the same email address to the [POST /identity/map](../endpoints/post-identity-map.md) endpoint at the same time, they would both get the same raw UID2 in response.
+
+However, there is a variable factor, which is the [salt](../ref-info/glossary-uid.md#gl-salt) value that's used in generating the raw UID2. The salt values are rotated periodically. If the salt value changes between one request and another, those two requests result in two different raw UID2, even when the DII is the same.
```
Comment (sunnywu) [inline docs/getting-started/gs-faqs.md:201]: Shall we add a link to the integration guide where salt bucket is discussed as "For further info"?
https://unifiedid.com/docs/guides/advertiser-dataprovider-guide#3-monitor-for-salt-bucket-rotations-related-to-your-stored-raw-uid2s

File: docs/getting-started/gs-faqs.md [hunk 2]
```diff
@@ -33,6 +34,12 @@ No. UID2 has its own framework, which is separate from EUID. As such, paperwork
 
 Yes. Through the [Transparency and Control Portal](https://www.transparentadvertising.com/), users can opt out from being served targeted ads tied to their UID2 identity. Each request is distributed through the UID2 Opt-Out Service, and UID2 Operators make the opt-out information available to all relevant participants.
 
+#### When I send DII to UID2, does UID2 store the information?
+
+No, UID2 does not store any DII. In addition, in almost all cases, UID2 doesn't store any values at all once the [POST&nbsp;/token/generate](../endpoints/post-token-generate.md), [POST&nbsp;/token/refresh](../endpoints/post-token-refresh.md), or [POST /identity/map](../endpoints/post-identity-map.md) call is complete.
+
+A necessary exception is the case where a user has opted out. In this scenario, UID2 stores a hashed, opaque value to indicate the opted-out user. The stored value cannot be reverse engineered to recognize the DII, but can be used to identify future requests for a UID2 generated from the same DII, which are therefore denied.
```
Comment (sunnywu) [inline docs/getting-started/gs-faqs.md:41]: _cannot be reverse engineered to recognize the DII_ 

Is it clearer to say

_cannot be reverse engineered back to the original value of the DII_ 

(i don't understand the term _recognize_ here too)
?

File: docs/getting-started/gs-faqs.md [hunk 3]
```diff
@@ -185,6 +194,12 @@ When a user opts out of UID2-based targeted advertising through the [Transparenc
 
 If a user opts out through your website, you should follow your internal procedures for handling the opt-out. For example, you might choose not to generate a UID2 for that user.
 
+#### Does the same DII always result in the same raw UID2?
+xxx
```
[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/getting-started/gs-faqs.md:198]: Yikes. My bad. TYSM.

File: docs/getting-started/gs-faqs.md (hunk 1 above)
Comment (genwhittTTD) [reply docs/getting-started/gs-faqs.md:201]: @sunnywu Great suggestion. Done. Thx.

File: docs/getting-started/gs-faqs.md (hunk 2 above)
Comment (genwhittTTD) [reply docs/getting-started/gs-faqs.md:41]: @sunnywu that sounds great -- done -- thx.

**Pull request merged**